src/mlbstandings/app.py

```
@@ -0,0 +1,51 @@
+    authed_session = AuthorizedSession(creds)  # type: ignore
+    files = light_google_wrappers.Files(authed_session)
+    sheets = light_google_wrappers.Spreadsheets(authed_session)
+    base_web = web.Web()
+    w = AbstractRateLimitedWeb(base_web, SimpleRateLimiter(15))
+    u = updater.Updater(d, files, sheets, CONTENTS_SPREADSHEET_ID, w)
+    # TODO remove once everything works with new versions
+    base_web.read('https://www.baseball-reference.com/')
+    print('No problem reading www.baseball-reference.com')
+    while True:
+        status = u.update()
+        if status == None or status == updater.SeasonStatus.OVER or not backfill:
```

Flake8 (7.2.0)

[error] comparison to None should be 'if cond is None:'

(E711)